Common/src/grid_movement_structure.cpp

```diff
@@ -7871,11 +7940,24 @@ CFreeFormDefBox::CFreeFormDefBox(unsigned short val_lDegree, unsigned short val_
 			}
 		}
 	}
-	 
+
+  BlendingFunction = new CFreeFormBlending*[nDim];
```

Comment:
nDim is always initialized to 3 for this class, independent of the actual dimension.